app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,21 @@
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
```

Comment:
Add it to helpers

Reply:
added helpers to cover links partial logic